### PR TITLE
add go 1.12.x to the travis, circleci, and promu YAMLs

### PR DESCRIPTION
**Pull request by tariq1890**
New version, new commit.

### CODE REVIEW

**Comment by roidelapluie** [comment]
You could also update promu and circleci:

https://github.com/prometheus/alertmanager/pull/1782/files

**Comment by tariq1890** [comment]
Thanks! Will do :)

**Comment by tariq1890** [comment]
PR title and commits updated

**Comment by roidelapluie** [comment]
Do we want tls 1.3? https://github.com/containous/traefik/blob/fb617044e0221b7f9f0665d4c8adaa3736335cf4/cmd/traefik/traefik.go#L46-L52

File: .travis.yml [hunk 1]
```diff
@@ -4,6 +4,7 @@ language: go
 # should also be updated.
 go:
 - 1.11.x
```
Comment (simonpasquier) [inline .travis.yml]: we don't support multiple versions of Go so 1.11.x needs to be removed.

**Comment by simonpasquier** [comment]
Re TLS 1.3, it can still be enabled at runtime so I wouldn't hack something in the binary.

**Comment by tariq1890** [comment]
@simonpasquier I understand. If I were to do that, I would need to update documentation where they mention go1.11 or higher. Shall I do that as well?

**Comment by simonpasquier** [comment]
@tariq1890 go for it if you want to but I wouldn't be too much concerned. What we really require now is a Go version that supports modules.

**Comment by tariq1890** [comment]
@simonpasquier Then I can just continue with these changes. I'd say my PR is ready for review.

**Comment by tariq1890** [comment]
Friendly Ping!

**Comment by simonpasquier** [comment]
The first RC of v2.8.0 has been announced yesterday so we're technically merging bug fixes only. Personally, I'd refrain on bumping the Go version now but @codesome will have the last word.

**Comment by tariq1890** [comment]
Got it. I completely understand.

**Comment by codesome** [comment]
I would also refrain from bumping the go version for v2.8.0.

**Comment by tariq1890** [comment]
@simonpasquier @codesome Can this be merged now that v2.8.0 is out?

**Comment by simonpasquier** [comment]
LGTM

**Comment by tariq1890** [comment]
Pinging @codesome

**Comment by brian-brazil** [comment]
Thanks!

The release is out by now.

**Pull request merged**